src/Microsoft.ML/Data/TextLoader.cs

```diff
@@ -0,0 +1,180 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using System;
+using System.Linq;
+using System.Reflection;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.ML.Data
+{
+    public sealed partial class TextLoaderRange
+    {
+        public TextLoaderRange()
+        {
+        }
+
+        /// <summary>
+        /// Convenience constructor for the scalar case.
+        /// Min and Max are set to the single value ordinal.
+        /// When a given column spans only a single column in the
+        /// dataset.
```

Comment:
You're using two distinct meanings of the word column in one sentence. This is fairly confusing.